### PR TITLE
Project done! Except challenge4

### PR DESCRIPTION
**Pull request by AhmadJer99**
It was nice and not hard to follow the documentation, I spaghetti tangled my code a little bit with my project references, first time using them, I  wasn't 100% sure what I was doing but I tried. 
Anyways the code is functional and you may review  and test it.

### CODE REVIEW

**Review by arashi256 — APPROVED** [review]
Hi, good job! I like the way you've used Spectre to enforce input validation and everything works well! I would consider having an easy and tidy way to quit the application from the main menu - I could only exit nicely after performing an operation and had to use control-c from the main menu. But this is a minor thing :) See you on the next project!

**Comment by AhmadJer99** [comment]
> Hi, good job! I like the way you've used Spectre to enforce input validation and everything works well! I would consider having an easy and tidy way to quit the application from the main menu - I could only exit nicely after performing an operation and had to use control-c from the main menu. But this is a minor thing :) See you on the next project!

hey, Thanks for the feedback ! much preciated ! yes i will work on two things in the app asap , which are 
trying to tidy up my application insides by applying some coding principles such ass KISS and SOLID and other principles, and i will implement a CloseApplication feature in this console app hopefully the correct way instead of ctrl+c'ing
